packages/expo-image/src/Image.tsx

```diff
@@ -27,6 +27,7 @@ export interface ImageProps extends AccessibilityProps {
   source?: ImageSourcePropType | null;
   style?: StyleProp<ImageStyle>;
   resizeMode?: ImageResizeMode;
+  blurRadius?: number;
```

Comment:
Let's annotate it's Android only for now:
```suggestion
  /**
   * @Android only
   */
  blurRadius?: number;
```